olp-cpp-sdk-core/include/olp/core/context/Context.h

```diff
@@ -170,7 +170,32 @@ class CORE_API Context {
    * @return The `android.content.Context` instance.
    */
   static jobject getAndroidContext();
-#endif
+#elif defined(__APPLE__)
+  /**
+   * @brief Inform subscribers to enter background mode.
+   *
+   * @note Use it only after you initialize the `Context` class.
+   */
+  static void EnterBackground();
+
+  /**
+   * @brief Inform subscribers to exit background mode.
+   *
+   * @note Use it only after you initialize the `Context` class.
+   */
+  static void ExitBackground();
+
+  /**
+   * @brief Store background sessions completion handler to call it
+   * when the session is done.
+   *
+   * @note Use it only after you initialize the `Context` class.
```

Comment:
Could you add params details, what is completion handler is and why it is needed, etc.?
Variable names are not according to coding style.

Reply:
done